website/notifications/emails.py

```diff
@@ -87,7 +88,8 @@ def store_emails(recipient_ids, notification_type, event, user, node, timestamp,
     if notification_type == 'none':
         return
 
-    template = event + '.html.mako'
+    # Reviews has three email templates for the global_reviews event.
+    template = context['template'] + '.html.mako' if event == 'global_reviews' else event + '.html.mako'
```

Comment:
I don't think we should have logic that depends on specific event names. Instead, maybe do something like `context.get('template', event)`, allowing anyone to pass in a custom template if they wish. Or, to be safer in case anyone ever actually has a `'template'` key in their context, add `template=None` as a parameter to `notify`.